src/vorta/utils.py

```diff
@@ -84,6 +84,8 @@ def get_private_keys():
     available_private_keys = []
     if os.path.isdir(ssh_folder):
         for key in os.listdir(ssh_folder):
+            if not os.path.isfile(os.path.join(ssh_folder, key)):
```

Comment:
there is another join() some lines below, so it would be better to just compute it once.